lms/djangoapps/organizations/views.py

```diff
@@ -152,6 +153,30 @@ def users(self, request, pk):
 
                     response_data.append(user_data)
             return Response(response_data, status=status.HTTP_200_OK)
+        elif request.method == 'DELETE':
+            user_ids = request.DATA.get('users')
+            if not user_ids:
+                return Response({"detail": _('users parameter is missing.')}, status.HTTP_400_BAD_REQUEST)
+            try:
+                user_ids = map(int, user_ids.split(','))
```

Comment:
`map(int, user_ids.split(','))
`
would throw an exception if a user supplies the list with a trailing comma - leaving the user wondering what was wrong. The exception would be thrown because the `split()` method would create a None item in the end of the list - and int(None) would raise an exception.

Using the `filter()` method would remove empty elements.
map(int, filter(None, user_ids.split(',')))

nit: user_ids = [ int(id) for id in filter(None, user_ids.split(',')) ] might be more pythonic.